### DISCUSSION TITLE
[Feature] [Nodejs] Support Node18 native fetch

### DISCUSSION THREAD

**Discussion by marklai1998** [apache/skywalking#9826] [Ideas]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Now the Fetch API is officially built in in Node18, its great to support it buit in in the skywalking-js

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Which component are you asking? UI, browser agent and Nodejs server agents are Node version relative.

**Comment by wu-sheng** [comment]
And issue is only for issues having people working on. Otherwise, this should only be on discussion panel.
